### PR TITLE
Bug/fix incorrect begin time

### PR DESCRIPTION
**Pull request by ToshY**
Fixes PR #178 from initial issue #174

### CODE REVIEW

File: entrypoint [hunk 1]
```diff
@@ -172,8 +172,8 @@ else
     waittime=$(( ${BASH_REMATCH[1]} * 60 ))
     target_time=$(($current_time + $waittime))
   else
-    today=$(date +"%Y%m%d")
-    target_time=$(date --date="${today}T${DB_DUMP_BEGIN}" +"%s")
+    today=$(date +"%Y-%m-%d")
```
Comment (deitch) [inline entrypoint:175]: This line is good.

File: entrypoint [hunk 2]
```diff
@@ -172,8 +172,8 @@ else
...
+    target_time=$(date --date="${today} ${DB_DUMP_BEGIN}" +"%s")
```
Comment (deitch) [inline entrypoint:176]: But let's leave the `T` here. If you try it out, you will see that it works with the `T`.

**Review by deitch — CHANGES_REQUESTED** [review]
Let's keep the `T`, as it works.

Also, please squash to a single commit.

File: entrypoint (hunk 2 above)
Comment (ToshY) [reply entrypoint:176]: @deitch Well I tried it out and I get the following:

Using `YY-MM-DDThhmm`
```
echo $(date --date="2021-12-14T0130" +"%s")
date: invalid date ‘2021-12-14T0130’
```

Using `YY-MM-DD hhmm`
```
echo $(date --date="2021-12-14 0130" +"%s")
1639441800
```

Comment (deitch) [reply entrypoint:176]: I _know_ I had it working before, but now I cannot reproduce it. It does work with `"2021-12-14T01:30"` but not without the `:`

Comment (deitch) [reply entrypoint:176]: Well, as long as `DB_DUMP_BEGIN` is without the `:`, we will have to live with it like that, and go without the `T`

**Comment by deitch** [comment]
So just the strange first commit you have

**Pull request closed without merging**